```
Error: processor_test.go:60:6: func `parseTime` is unused (unused)
make[2]: *** [../../Makefile.Common:96: lint] Error 1
make[1]: *** [Makefile:163: for-all-target-./processor/logstransformprocessor] Error 2
make[1]: *** Waiting for unfinished jobs....
func parseTime(format, input string) *time.Time {
     ^
Error: processor_test.go:36:2: var `cfg` is unused (unused)
	cfg = &Config{
	^
```

--- a/processor/logstransformprocessor/go.mod
+++ b/processor/logstransformprocessor/go.mod
@@ -3,8 +3,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/processor/logst
 go 1.17
 
 require (
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.47.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza v0.49.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.50.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza v0.50.0
 	github.com/open-telemetry/opentelemetry-log-collection v0.29.1
 	github.com/stretchr/testify v1.7.1
 	go.opentelemetry.io/collector v0.50.1-0.20220429151328-041f39835df7

--- a/processor/logstransformprocessor/processor_test.go
+++ b/processor/logstransformprocessor/processor_test.go
@@ -74,8 +74,13 @@ type testLogMessage struct {
 	attributes   *map[string]pdata.Value
 }
 
-func TestLogsTransformProcessor(t *testing.T) {
+// Skips test without applying unused rule: https://github.com/dominikh/go-tools/issues/633#issuecomment-606560616
+var skip = func(t *testing.T) {
 	t.Skip("Flaky test - See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9761")
+}
+
+func TestLogsTransformProcessor(t *testing.T) {
+	skip(t)
 	baseMessage := pcommon.NewValueString("2022-01-01 INFO this is a test")
 	spanID := pcommon.NewSpanID([8]byte{0x32, 0xf0, 0xa2, 0x2b, 0x6a, 0x81, 0x2c, 0xff})
 	traceID := pcommon.NewTraceID([16]byte{0x48, 0x01, 0x40, 0xf3, 0xd7, 0x70, 0xa5, 0xae, 0x32, 0xf0, 0xa2, 0x2b, 0x6a, 0x81, 0x2c, 0xff})

